test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/KieServerWithWorkbenchSurvivalIntegrationTest.java

```diff
@@ -0,0 +1,170 @@
+        assertThat(queryServicesClient.findProcessInstances(0, 10)).isNotNull().hasSize(1);
+
+        scaleKieServerTo(0);
+        checkServerTemplateInstanceCount(serverInfo.getServerId(), 0);
+
+        logger.debug("Send REST request to Kie Server");
+        sendRESTCallsToUnavailableKieServer();
+
+        scaleKieServerTo(1);
+        checkServerTemplateInstanceCount(serverInfo.getServerId(), 1);
+
+        logger.debug("Start new process instance");
+        Long newPid = processServicesClient.startProcess(CONTAINER_ID, SIGNALTASK_PROCESS_ID);
+        assertThat(newPid).isNotNull().isGreaterThan(0L);
+
+        logger.debug("Check started processes");
+        assertThat(queryServicesClient.findProcessInstances(0, 10)).isNotNull().hasSize(2);
+
+        ProcessInstance processInstance = processServicesClient.getProcessInstance(CONTAINER_ID, signalPid);
+        assertThat(processInstance).isNotNull();
+        assertThat(processInstance.getState()).isNotNull().isEqualTo(org.kie.api.runtime.process.ProcessInstance.STATE_ACTIVE);
+    }
+
+    private void scaleKieServerTo(int count) {
+        logger.debug("Scale Kie Server to " + count);
+        deploymentScenario.getKieServerDeployment().scale(count);
+        deploymentScenario.getKieServerDeployment().waitForScale();
+    }
+
+    protected void checkServerTemplateIsRegistred(String expectedId, String expectedName) {
+        assertThat(kieServerMgmtControllerClient.listServerTemplates()).hasSize(1);
+        ServerTemplate serverTemplate = kieServerMgmtControllerClient.listServerTemplates().iterator().next();
+        assertThat(serverTemplate.getId()).isEqualTo(expectedId);
+        assertThat(serverTemplate.getName()).isEqualTo(expectedName);
+    }
+
+    protected void checkServerTemplateInstanceCount(String serverTemplateId, int expected) {
+        ServerTemplate serverTemplate = kieServerMgmtControllerClient.getServerTemplate(serverTemplateId);
+        assertThat(serverTemplate).isNotNull();
+        assertThat(serverTemplate.getServerInstanceKeys()).isNotNull().hasSize(expected);
+    }
+
+    private void sendRESTCallsToUnavailableKieServer() {
+        Client httpKieServerClient = new ResteasyClientBuilder()
+                .establishConnectionTimeout(10, TimeUnit.SECONDS)
+                .socketTimeout(10, TimeUnit.SECONDS)
+                .register(new Authenticator(deploymentScenario.getKieServerDeployment().getUsername(),
+                                deploymentScenario.getKieServerDeployment().getPassword()))
+                .build();
+
+        WebTarget target = httpKieServerClient.target(deploymentScenario.getKieServerDeployment().getUrl().toString() + "/services/rest/server/containers");
+        Response response = target.request().get();
+        response.close();
+        assertThat(response.getStatus()).isEqualTo(Response.Status.SERVICE_UNAVAILABLE.getStatusCode());
+
+        URL postUrl = null;
```

Comment:
This part with POST request isn't needed as the previous GET call checks that Kie server is unavailable.